File: hello_world.rb
puts "How old are you?"
age = gets.chomp.to_i
if age == 0
	puts "nope"
	exit
end
puts "You are #{age} years old"
if age < 30
	puts "you're still a baby!"
end
if age > 30
	puts "golly, you're a fossil!"
end

File: hello world.cpp
// hello world
#include<iostream> 
using namespace std;
int main()
{
	int age;
	cout << "How old are you?";
	cin >> age;
	
	if (age==0)
		{
		cout << "nope";
		return 0;
		}
	cout << "You are ";
	cout << age ;
	cout << " years old.";
	if (age<30) 
	cout << " your still a baby!";
	if (age>30) 
	cout<< " golly, you're a fossil!";
	return 0;	
}
